canonical-websites/www.ubuntu.com
# 3338 ai index page

## Done

Add `/ai` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/ai>
- See that it matches the [copy doc](https://docs.google.com/document/d/1mJq8PxH8FVRVF9i1Xzh0f6M73eyjqZE4NaKYHRcRAKM/edit#)


## Issue / Card

Fixes #3338 
Fixes #3466
